### PR TITLE
Update module name

### LINKED ISSUE #1: Update module name
The [module name](https://github.com/conduitio-labs/conduit-connector-google-cloudstorage/blob/197d510101d8d9b6f4986174695f0541b3e79abe/go.mod#L1) should be `github.com/conduitio-labs/conduit-connector-google-cloudstorage`, also search and update any references in the code to the module (e.g. I found [this import](https://github.com/conduitio-labs/conduit-connector-google-cloudstorage/blob/197d510101d8d9b6f4986174695f0541b3e79abe/cmd/connector/main.go#L20)).

### PR DESCRIPTION
**Pull request by hariso**
### Description

Fixes #1.

### Quick checks:

- [x] I have followed the [Code Guidelines](https://github.com/ConduitIO/conduit/blob/main/docs/code_guidelines.md).
- [x] There is no other [pull request](https://github.com/ConduitIO/conduit-connector-s3/pulls) for the same update/change.
- [ ] I have written unit tests.
- [ ] I have made sure that the PR is of reasonable size and can be easily reviewed.

### CODE REVIEW

File: spec.go [hunk 1]
```diff
@@ -39,7 +39,7 @@ func specification() sdk.Specification {
 		Summary:     "An Google Cloud Storage Source and Destination Connector for Conduit, Written in Go.",
 		Description: "Real time data transmission with google cloud storage",
 		Version:     version,
-		Author:      "Santosh Kumar Gajawada",
+		Author:      "InfraCloud Technologies & Meroxa, Inc.",
```
Comment (maha-mstfa) [inline spec.go:40]: oh you got the update too, you can overwrite the one I did, "&" looks nicer

**Review by maha-mstfa — APPROVED** [review]
lgtm

File: spec.go (hunk 1 above)
Comment (hariso) [reply spec.go:40]: OK, I'll do that. Maybe you merge your PR first, and then I merge this one, with the overwrite?

Comment (maha-mstfa) [reply spec.go:40]: yes this is what I meant.. my PR is merged now

**Pull request merged**